docs/source/_json/controlpanels_get.resp

```diff
@@ -0,0 +1,15 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
```

Comment:
@jaroel how do you make it possible that you just show the mail and editing control panel and not all available control panels?

Reply:
The available controlpanels in the rest API are define in the service/controlpanels/configure.zcml, as most -but not all- controlpanels are registry based.

Reply:
@jaroel that's why I am asking, we have more than two controlpanels listed there and just two show up in the response.